plugins/inputs/kube_inventory/kube_inventory.go

```diff
@@ -159,6 +161,8 @@ var (
 	podContainerMeasurement          = "kubernetes_pod_container"
 	serviceMeasurement               = "kubernetes_service"
 	statefulSetMeasurement           = "kubernetes_statefulset"
+	resourcequotaMeasurement         = "kubernetes_resourcequota"
+	secretMeasurement                = "kubernetes_secret" //nolint:gosec // false positive
```

Comment:
Is there a reason you chose "secret" over "certificates"?
Can we get an example in the readme as well.

Reply:
Do you mean the name here? To be changed from **kubernetes_secret** to **kubernetes_certificate**? If yes then surely we can do it. 
The reason why I needed to add this is to identify the tls certificates available as secrets in k8s and get their expiry. For now I am extracting only **tls.crt** data key, but this can be extended to others as well, depending on requirements. Before I was parsing all the data keys, but in our infrastructure monitoring, we were getting multiple keys of same value.
We are using x509 for non k8s resources, but that means we need to know the path of it, while c.CoreV1().Secrets is an abstraction that gives all secret types including certificates.

Reply:
thanks for the background - let's rename to certificate as well